platform/ui/src/components/DisplaySetMessageListTooltip/DisplaySetMessageListTooltip.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Tooltip from '../Tooltip';
+import Icon from '../Icon';
+import { useTranslation } from 'react-i18next';
+
+const DisplaySetMessageListTooltip = ({ messages }): React.ReactNode => {
+  const { t } = useTranslation('Messages');
+  if (messages.size()) {
+    return (
+      <div>
+        <Tooltip
+          position="left"
+          tight={true}
+          content={
+            <div className="text-left max-w-40">
```

Comment:
is it series or displaySet messages

Reply:
I dont know if the OHIF users, not programmers, knows about what is displaySet, but I can change if you want

Reply:
DisplaySet is a DICOM construct, see what is done in OHIF v2